src/util/system.cpp

```diff
@@ -935,6 +935,20 @@ static bool GetConfigOptions(std::istream& stream, const std::string& filepath,
     return true;
 }
 
+bool IsConfSupported(KeyInfo& key, std::string& error) {
```

Comment:
As this function is used in the only compilation unit it could be declared within an anonymous `namespace` or as `static` to keep global namespace hygiene (the former option is my personal preference :tiger2: ).

See: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities